src/Samples/VisualBasic/ConsoleClassifier/ConsoleClassifierVB.vbproj

```diff
@@ -48,39 +50,10 @@
     </ProjectReference>
   </ItemGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
-    <DebugSymbols>true</DebugSymbols>
-    <DefineDebug>true</DefineDebug>
-    <DefineTrace>true</DefineTrace>
-    <OutputPath>bin\Debug\</OutputPath>
-    <NoWarn>$(NoWarn);42016,41999,42017,42018,42019,42032,42036,42020,42021,42022</NoWarn>
     <DefineConstants>$(DefineConstants),_MyType="Empty"</DefineConstants>
```

Comment:
@jaredpar, is this 'by-design'? We define `MyType=Console` globally, but override it for Debug (and only debug) to be `_MyType=Empty`.